internal/juju/secrets.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the Apache License, Version 2.0, see LICENCE file for details.
+
+package juju
+
+import (
+	"errors"
+	"fmt"
+	jujuerrors "github.com/juju/errors"
+	"github.com/juju/juju/api"
+	apisecrets "github.com/juju/juju/api/client/secrets"
+	coresecrets "github.com/juju/juju/core/secrets"
```

Comment:
There should be 2 stanzas of inputs here

Reply:
done